modules/run/artifacts/opt/jboss/container/java/run/run-java.sh

```diff
@@ -96,13 +103,8 @@ load_env() {
   if [ -z "${JAVA_APP_DIR}" ]; then
```

Comment:
this existing code is never executed: the run_env.sh code above will set JAVA_APP_DIR if it unset. Therefore this if branch will always be false. Since you're working in this area, please delete this if. The `else` branch, below, should just be run unconditionally.